doc/cli.md

```diff
@@ -126,10 +126,10 @@ Option | Description
 `--input`, `-i` (required) | Path to input profile.
 `--map`, `-m` (required) | Path to mapping of source features to target features.
 `--output`, `-o` (required) | Path to output profile.
-`--frac`, `-f` | Count each target feature as 1 / _k_ (_k_ is the number of targets mapped to a source). Otherwise, count as one.
+`--divide`, `-d` | Count each target feature as 1 / _k_ (_k_ is the number of targets mapped to a source). Otherwise, count as one.
```

Comment:
Check --frac parameter. Previously was --normalize, and is now inconsistent within the codebase. 

--frac actually does sound more intuitive than --divide.

Reply:
It was modified during a previous effort in replacing Travis CI with GitHub Actions (I changed some trivial code in order to fire a PR...). Now it should be consistently `--divide`.